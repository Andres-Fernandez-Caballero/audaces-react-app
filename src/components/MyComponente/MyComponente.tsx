import React from 'react';

export interface MyComponenteProps {}

const MyComponente: React.FunctionComponent<MyComponenteProps> = ({}) => {
	return <>MyComponente</>;
};

export default MyComponente;
