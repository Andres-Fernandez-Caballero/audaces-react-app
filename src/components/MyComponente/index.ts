export { default } from './MyComponente';
export * from './MyComponente';
