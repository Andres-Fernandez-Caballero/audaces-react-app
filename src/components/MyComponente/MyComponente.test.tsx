import { cleanup, render } from '@testing-library/react';
import React from 'react';
import MyComponente, { MyComponenteProps } from './MyComponente';

describe('MyComponente', () => {
	afterEach(cleanup);
	const defaultProps: MyComponenteProps = {};

	it('should render', () => {
		const props = { ...defaultProps };
		const { asFragment, queryByText } = render(<MyComponente {...props} />);

		expect(asFragment()).toMatchSnapshot();
		expect(queryByText('MyComponente')).toBeTruthy();
	});
});
