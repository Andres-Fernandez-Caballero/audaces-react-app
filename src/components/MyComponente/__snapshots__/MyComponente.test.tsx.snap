// Vitest Snapshot v1

exports[`MyComponente > should render 1`] = `
<DocumentFragment>
  MyComponente
</DocumentFragment>
`;
